app/src/main/java/io/apicurio/registry/content/canon/GraphQLContentCanonicalizer.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2019 Red Hat
```

Comment:
2020 :-)

Reply:
Damn you, copy/paste!! :)

Reply:
I realized I need a unit test for this anyway - so I'll add a unit test and fix the (c) year.